graph/schema.graphqls

```diff
@@ -274,6 +279,15 @@ type AftermarketDevice implements Node {
   The Image Url of the device
   """
   image: String!
+  """
+  The earnings attached to the aftermarket device
+  """
+  earnings(
```

Comment:
We seem to have gone against what we did for `Vehicle`, where the "big" resolver is one level deeper.

Reply:
I felt since this works with a single resolver, it makes things simpler, rather than using the multiple resolvers like we had in the vehicle endpoint.

Reply:
Doing it this way breaks from the Relay spec and makes our API non-uniform. I don't think a slightly simpler implementation is a strong enough argument in the other direction.